.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "eslint.workingDirectories": [
+    "packages/example-react-app",
+    "packages/web3-plugin-wallet-rpc"
+  ]
+}
```

Comment:
Should this file be checked-in?

Reply:
ok, removed for now, let's wait if others will have similar problems